server/index.js

```diff
@@ -80,6 +80,13 @@ export default class Server {
       const p = join(__dirname, '..', 'client', ...(params.path || []))
       await serveStatic(req, res, p)
     })
+
+    this.router.get('/_webpack/:number', async (req, res, params) => {
```

Comment:
I also am not sure exactly when this path is used, but some files, like patch files of HMR, are served from it.